config/README.md

```diff
@@ -1,102 +1,103 @@
-# Multi-region cockroachdb setup
+# Deploying a DSS
+
+## Preface
+
+This doc provides a well-lit path for deploying the DSS and its dependencies (
+namely CockroachDB) on Kubernetes. This is not a requirement, and a DSS node
+can join a cluster as long as it meets the following requirements:
+
+*   Every CockroachDB node must advertise a unique and routable address.
+    *   It is preferred to use domain names with unique prefixes and homogenous
+        suffixes, ie: 0.c.dss.interussplatform.com.
+    *   This allows wildcard usage in the CRDB certificates.
+*   Every DSS node should run a minimum of 3 CockroachDB nodes.
```

Comment:
Let's elaborate on the reasoning behind this recommendation so that others can identify when (if ever) it might not apply.

Reply:
which bullet point are specifically commenting on here?

Reply:
added for the min CRDB nodes